demo/deploy-hcp-bicep.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Create an ARO HCP Cluster + Node pool using bicep.
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source env_vars
+LOCATION=uksouth # The ONLY supported region for ARO-HCP in INT
+
+# This is the only supported subscription for creating INT hcp/nodepools
+az account set -n "ARO SRE Team - INT (EA Subscription 3)" 
```

Comment:
This could be an unexpected side-effect of running the script.  Instead of changing the default can we just use `--subscription` in all the `az` commands?